### PR TITLE
revise definitions of Hard palate and Palatine bone

### LINKED ISSUE #2603: palate bone, palatine bone, and hard palate
These are confusing and adding more information in the definition would help.

Hard palate: change "and palatine" to "and the palatine bone".
Palatine bone: add that it helps for the hard palate.

palate bone: it is unclear to me how this is different from palatine bone.

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #2603

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@megbalk would you like to review this PR?

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -28294,7 +28294,7 @@ property_value: taxon_notes "In primitive bony fish and tetrapods, the nasal bon
 [Term]
 id: UBERON:0001682
 name: palatine bone
-def: "A paired dermal bone situated in the mid-palate[Palaeos]." [http://palaeos.com/vertebrates/bones/dermal/palatal-palatines.html, Wikipedia:Palatine_bone]
+def: "A paired dermal bone situated in the mid-palate that helps for the hard palate [Palaeos]." [http://palaeos.com/vertebrates/bones/dermal/palatal-palatines.html, Wikipedia:Palatine_bone]
```
Comment (megbalk) [inline src/ontology/uberon-edit.obo:28297]: "for" should be "form".

**Review by megbalk — CHANGES_REQUESTED** [review]
There is a misspelling (propagated from my issue request) in the def for palatine bone.

File: src/ontology/uberon-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [inline src/ontology/uberon-edit.obo:28297]:
```suggestion
def: "A paired dermal bone situated in the mid-palate that helps form the hard palate [Palaeos]." [http://palaeos.com/vertebrates/bones/dermal/palatal-palatines.html, Wikipedia:Palatine_bone]
```

Comment (nicolevasilevsky) [reply src/ontology/uberon-edit.obo:28297]: fixed

**Review by megbalk — APPROVED** [review]

**Pull request merged**